packages/blaze/builtins.js

```diff
@@ -89,13 +137,35 @@ Blaze.Each = function (argFunc, contentFunc, elseFunc) {
   eachView.contentFunc = contentFunc;
   eachView.elseFunc = elseFunc;
   eachView.argVar = new ReactiveVar;
+  eachView.variableName = null;
+
+  // update the @index value in the scope of all subviews in the range
+  var updateIndexes = function (from, to) {
+    if (to === undefined) {
+      to = eachView.numItems - 1;
+    }
+
+    for (var i = from; i <= to; i++) {
+      var view = eachView._domrange.members[i].view;
+      var reactiveVar = view._scopeBindings['@index'];
+      reactiveVar.set(i);
```

Comment:
It is not possible if nothing interferes with Blaze because Blaze controls what views are added to that range. I will rewrite the line to not create a separate variable.